wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/CodenvyLicenseManagerObservable.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.api.license.server;
+
+/**
+ * @author Anatolii Bazko
+ */
+public interface CodenvyLicenseManagerObservable {
```

Comment:
Most of libraries from **java.utils** packages have more functionality that we really need. And it doesn't lid to writing own components instead. Moreover - using well-known classes helps to understand the code quickly.